tests/test_operation.py

```diff
@@ -1197,3 +1197,107 @@ def test_qubit_state_vector_decomposition(self, monkeypatch):
         assert len(call_args) == 1
         assert np.array_equal(call_args[0][0], state)
         assert np.array_equal(call_args[0][1], wires)
+
+class TestChannel:
```

Comment:
As Antal pointed out, we should be testing only aspects that are not already tested by `Operation`, in this case the two new methods. Unless I'm missing something, the tests should be straightforward: just check that the correct matrices are returned. It may be worth consulting with an expert, but I believe this will require making a "dummy" channel inside the tests

Reply:
Those tests will have to be in the next PR where I add the known noisy channels.